elements/rh-badge/rh-badge.ts

```diff
@@ -17,7 +30,6 @@ export class RhBadge extends BaseBadge {
 
   /**
    * Denotes the state-of-affairs this badge represents
```

Comment:
we should specify the semantic meaning of each option here
```suggestion
   * Denotes the state-of-affairs this badge represents
   * - **info**: yadda yadda
   * - **success**: etc etc
```

Need @coreyvickery's design input on that